### DISCUSSION TITLE
AWS Distributed Deployment sensor not connecting

### DISCUSSION THREAD

**Discussion by out-n-about** [Security-Onion-Solutions/securityonion#2531] [General] [answered]
Hello,

I'm setting up a distributed SO instance in AWS cloud to test with.  I have both instances in the same VPC and subnet and allowed them to communicate with each other in AWS Security Groups. 

The Manager installed fine as a ManagerSearch node.  I installed the sensor as sensor only.  (not sure if this is the best way to deploy this)

Once I start the install, provide manager name, IP, etc and it tried to connect I get the following message.  Any Ideas on a fix for this?

I verified that the file 2.3.21.tar.gz exists in the directory its looking for.  

![image](https://user-images.githubusercontent.com/24942018/103581332-ccf54e00-4ea9-11eb-8c9b-8d2252fa4315.png)

**Comment by weslambert** [comment, marked as answer]
You have to enable password auth beforehand (for the `soremote` user) when running in AWS, as it expects by default to use key auth.

**Reply by out-n-about** [reply]
That worked!  Thanks
